### PR TITLE
SLVSCODE-123 Pass Java vmLocation to the language serveur

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

**Review by jblievremont — APPROVED** [review]
LGTM :+1: tested with Oracle JDK 8 and OpenJDK 11

File: src/java.ts [hunk 1]
```diff
@@ -101,15 +101,20 @@ export async function getJavaConfig(
         return javaConfigDisabledInLightWeightMode();
       }
       const isTest: boolean = await extensionApi.isTestFile(fileUri);
-      const sourceLevel: string = (
-        await extensionApi.getProjectSettings(fileUri, ['org.eclipse.jdt.core.compiler.compliance'])
-      )['org.eclipse.jdt.core.compiler.compliance'];
+      const projectSettings: { [name: string]: string } = await extensionApi.getProjectSettings(fileUri, [
+        'org.eclipse.jdt.core.compiler.compliance',
+        'org.eclipse.jdt.ls.core.vm.location'
```
Comment (jblievremont) [inline src/java.ts:106]: Nitpick: maybe extract these 2 strings to constants

**Review by jblievremont — APPROVED** [review]
LGTM, tested on a Maven project with Oracle Java 8 and OpenJDK 11

File: src/java.ts (hunk 1 above)
Comment (henryju) [reply src/java.ts]: :+1:

**Pull request merged**